core-services/prow/02_config/openshift/ibm-powervs-block-csi-driver/_prowconfig.yaml

```diff
@@ -0,0 +1,113 @@
+tide:
+  queries:
+  - includedBranches:
+    - community-4.6
+    - community-4.7
+    - openshift-4.10
+    - openshift-4.2
+    - openshift-4.3
+    - openshift-4.4
+    - openshift-4.5
+    - openshift-4.6
+    - openshift-4.7
+    - openshift-4.8
+    - openshift-4.9
+    - release-4.0
+    - release-4.10
+    - release-4.2
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    - release-4.6
+    - release-4.7
+    - release-4.8
+    - release-4.9
+    labels:
+    - approved
+    - backport-risk-assessed
+    - bugzilla/valid-bug
+    - cherry-pick-approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - needs-rebase
+    repos:
+    - openshift/ibm-powervs-block-csi-driver
+  - includedBranches:
+    - openshift-4.11
+    - release-4.11
+    labels:
+    - approved
+    - bugzilla/valid-bug
+    - lgtm
+    - staff-eng-approved
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - needs-rebase
+    repos:
+    - openshift/ibm-powervs-block-csi-driver
+  - includedBranches:
+    - main
+    - master
+    labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
+    - do-not-merge/hold
+    - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - keep-main-query-separate
+    - needs-rebase
+    repos:
+    - openshift/ibm-powervs-block-csi-driver
+  - excludedBranches:
```

Comment:
what are these branched we are excluding?

Reply:
Discussed with @Karthik-K-N and added a minimalized prow config.